### PR TITLE
Handle connector configuration and metadata properly, add refresh token

### PR DESCRIPTION
**Pull request by lyuboxa**
Connector configuration is write-only and API will return an empty response when read back.
This is deliberate and will be address in future platform releases.

Refresh tokens can be used to acquire new access tokens which have expired, the parameter
is not required and can be omitted.

### CODE REVIEW

File: meroxa/resource_connector.go [hunk 1]
```diff
@@ -133,17 +135,6 @@ func resourceConnectorCreate(ctx context.Context, d *schema.ResourceData, m inte
 		input.PipelineName = v.(string)
 	}
 
-	if v, ok := d.GetOk("config"); ok {
-		input.Configuration = v.(map[string]interface{})
-	}
-
-	if v, ok := d.GetOk("input"); ok {
```
Comment (dianadoherty) [inline meroxa/resource_connector.go:140]: don't we still need input?

Comment (lyuboxa) [reply meroxa/resource_connector.go:140]: Yes, we do, it is set when the config is set.

**Review by raulb — APPROVED** [review]
LGTM

File: docs/index.md [hunk 2]
```diff
@@ -40,12 +40,12 @@ provider "meroxa" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- **access_token** (String, Sensitive)
```
Comment (terrancej) [inline docs/index.md:45]: optional?

Comment (lyuboxa) [reply docs/index.md:45]: ha yes.. wow

Comment (lyuboxa) [reply docs/index.md]: fixed it btw.

**Pull request merged**